docs/deploying.md

```diff
@@ -238,6 +238,16 @@ NAME             STATUS   MESSAGE
 openstack-edpm   False    Deployment not started
 ```
 
+### NodeSet Config Changes
+We create a Hash of the inputs located in the OpenStackDataPlaneNodeSet Spec Nodes and NodeTemplate sections. This hash is then stored in the `status.configHash` field. If the current value of the configHash is different to the deployed configHash, then we set `status.configChanged`:
+
+```
+❯ oc get osdpns -o yaml | yq '.items[].status.configChanged'
```

Comment:
It seems to be leftover from attempt in https://github.com/openstack-k8s-operators/dataplane-operator/pull/490

Reply:
Ahh true. Yes, I need to update the docs.